src/main/java/pl/touk/sputnik/configuration/GeneralOption.java

```diff
@@ -9,7 +9,10 @@
     PROCESS_TEST_FILES("global.processTestFiles", "Process test files?", "true"),
     MAX_NUMBER_OF_COMMENTS("global.maxNumberOfComments", "Maximum number of comments to submit", "0"),
     COMMENT_ONLY_CHANGED_LINES("global.commentOnlyChangedLines", "Comment only changed lines and context", "false"),
-
+    COMMENT_FORMAT("global.commentFormat", "Sputnik comment format. {0}: reporter, {1}: level, {2}: message",
```

Comment:
I'd like to have separate package for messages and formats. messages.commentFormat, messages.problemFormat would be a nice separation